dags/main_summary.py

```diff
@@ -215,23 +215,41 @@
     executor=GetDefaultExecutor(),
     dag=dag)
 
-main_summary_experiments = MozDatabricksSubmitRunOperator(
+main_summary_experiments = SubDagOperator(
     task_id="main_summary_experiments",
-    job_name="Experiments Main Summary View",
-    execution_timeout=timedelta(hours=10),
-    instance_count=5,
-    max_instance_count=40,
-    enable_autoscale=True,
-    instance_type="i3.2xlarge",
-    spot_bid_price_percent=50,
-    owner="ssuh@mozilla.com",
-    email=["telemetry-alerts@mozilla.com", "frank@mozilla.com", "ssuh@mozilla.com", "robhudson@mozilla.com"],
-    env=tbv_envvar("com.mozilla.telemetry.views.ExperimentSummaryView", {
-        "from": "{{ ds_nodash }}",
-        "to": "{{ ds_nodash }}",
-        "bucket": "{{ task.__class__.private_output_bucket }}"}),
-    uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/telemetry_batch_view.py",
-    dag=dag)
+    subdag=moz_dataproc_jar_runner(
+        parent_dag_name=dag.dag_id,
+        dag_name="main_summary_experiments",
+        job_name="Experiments_Main_Summary_View",
+        main_class="com.mozilla.telemetry.views.ExperimentSummaryView",
+        jar_urls=[
+            "https://s3-us-west-2.amazonaws.com/net-mozaws-data-us-west-2-ops-ci-artifacts"
+            "/mozilla/telemetry-batch-view/master/telemetry-batch-view.jar",
+        ],
+        jar_args=[
+            "--from={{ds_nodash}}",
+            "--to={{ds_nodash}}",
+            "--inbucket=gs://moz-fx-data-derived-datasets-parquet",
+            "--bucket=s3a://telemetry-parquet",
+        ],
+        cluster_name="main-experiments-{{ds_nodash}}",
+        image_version="1.3",
+        worker_machine_type="n2-highmem-8",
+        num_workers=40,
```

Comment:
Are we enabling autoscaling on dataproc jobs? The load for this job tends to vary based on what experiments are running at any given time so autoscaling makes a lot of sense for it.

Reply:
the dataproc stuff we have right now doesn't support autoscaling. i'll file a bug to attempt to add that if i can't migrate it to sql soon